vexim/config/httpheaders.php

```diff
@@ -2,4 +2,5 @@
   session_start();
   header("Cache-control: private"); // IE6 hack to back forms + BACK button work
   header("Content-Type: text/html; charset=utf-8");
+  print '<!DOCTYPE html>';
```

Comment:
We have files which include `httpheaders.php`, but issue further `header()` calls. If you want this doctype, you should add it before the `<html>` tag to the files that have that tag.